code/game/turfs/floor_attackby.dm

```diff
@@ -596,10 +595,10 @@ var/mining_in_progress = null // Define the variable out of any scopes and the a
 		var/mob/living/human/H = user
 		var/obj/item/weapon/reagent_containers/glass/extraction_kit/ET = C
 		if (ET.reagents.total_volume > 0)
-			H << "<span class = 'notice'>Empty \the [ET] first.</span>"
+			to_chat(H, SPAN_NOTICE("Empty \the [ET] first."))
 			return
 		if (istype(H))
-			visible_message("<span class = 'notice'>[user] carefully examine \the [src] with \the [C.name]...</span>", "<span class = 'notice'>You start to carefully examine \the [src] with \the [C.name].</span>")
+			visible_message(SPAN_NOTICE("[user] carefully examines \the [src] with \the [C.name]..."), SPAN_NOTICE("You start to carefully examine \the [src] with \the [C.name]."))
```

Comment:
This should be set to user.visible_message(

Reply:
result is the same